setup.py

```diff
@@ -20,7 +20,7 @@ def read(fname):
     keywords=('serialization', 'deserialization', 'json',
               'marshal', 'marshalling', 'schema', 'validation',
               'multiplexing', 'demultiplexing', 'polymorphic'),
-    install_requires=['marshmallow==3.0.0b9'],
+    install_requires=['marshmallow==3.0.0b12'],
```

Comment:
can this be a soft pin like I did in: https://github.com/maximkulkin/marshmallow-oneofschema/pull/18/files#diff-2eeaed663bd0d25b7e608891384b7298R23